kernel/mod_mgmt/Cargo.toml

```diff
@@ -13,6 +13,11 @@ qp-trie = "0.8.0"
 cstr_core = "0.2.3"
 rangemap = "0.1.13"
 const_format = "0.2.2"
+lz4_flex = { version = "0.9.3", default-features = false, optional = true }
+cpio_reader = { version = "0.1.0", optional = true }
+
+[features]
+mod_unpack = ["lz4_flex", "cpio_reader"]
```

Comment:
document what this feature is used for.

Also, is there a clearer name for the feature, such as "unarchive_boot_modules" or "extract_..."?  I'm open to something else, but to be nitpicky, `mod_unpack` is a bit vague to my ear.

Reply:
done, renamed it to "extract_boot_modules"